src/utils/validation.ts

```diff
@@ -1,8 +1,11 @@
 import { FormDigestInfo } from "./spo.js";
 
 export const validation = {
-  isValidGuidArray(guids: string[]): boolean {
-    return guids.every(guid => this.isValidGuid(guid));
+  isValidGuidArray(guidsString: string): boolean | string {
```

Comment:
This returns a boolean or a string. Like already implemented on the UPN version. But should we really want this? It feels like javascript all over again, where you would not be sure what the value is that's coming out of it. Also we're validating an array but giving back a comma-separated string... not sure... i feel like it's useful, but not very clear. I'd thin returning a response object with a `success` and `invalidGuid` array would be more clear.

Reply:
Regarding error handling, this is a cleaner approach in my eyes. When you submit a bunch of arrays where a couple of these are invalid, in the error message you'll see all the invalid GUIDs listed at once, which makes for a cleaner user experience.

Reply:
The point I'm making is more that you would not want to concern yourself with output to the user in this helper function. The function should just validate the guids. How it's returned to the user should be done in the command itself.

Anyway, as it's already implemented like this for UPNs, and we're moving to Zod anyway, I'll just ignore this for now.